src/components/Table.svelte

```diff
@@ -22,17 +22,11 @@
     let currentRow = undefined
     let extraContent = undefined
 
-    if (e.target.parentNode.classList.contains("title")) {
-      title = e.target.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.nextElementSibling
-    } else {
-      title = e.target.parentNode.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.parentNode.nextElementSibling
-    }
+    title = e.target.parentNode.parentNode
```

Comment:
Gracias for catching that. I remember changing this because something I needed wasn't working but I don't remember what it was now. I made this change and everything looks good 👍